mmv1/templates/terraform/examples/cloud_run_service_multiple_regions.tf.erb

```diff
@@ -0,0 +1,192 @@
+# Cloud Run service replicated across multiple GCP regions
+
+######################################
```

Comment:
while i really like these in the config, i'm not sure it's necessary for the example. how would you feel about removing these comments?

Reply:
SG, I removed them from the example.